ivcfmerge/batch.py

```diff
@@ -0,0 +1,36 @@
+from contextlib import ExitStack
+from queue import Queue
+from tempfile import NamedTemporaryFile
+
+from . import ivcfmerge
+
+
+def ivcfmerge_batch(input_paths, output_path, batch_size=1000):
+    if batch_size < 2 or batch_size > len(input_paths):
+        batch_size = len(input_paths)
+
+    if batch_size == len(input_paths):
+        batch = input_paths
+    else:
+        work_queue = Queue()
+        [work_queue.put(item) for item in input_paths]
```

Comment:
I see no benefit to use list comprehension here over simple for loop. In fact it will create a temporary list which contains `len(input_paths)` `None` elements.